sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_generated/aio/operations/_path_operations.py

```diff
@@ -135,8 +135,8 @@ async def create(  # pylint: disable=inconsistent-return-statements
         :param cpk_info: Parameter group. Default value is None.
         :type cpk_info: ~azure.storage.filedatalake.models.CpkInfo
         :keyword callable cls: A custom type or function that will be passed the direct response
-        :return: None, or the result of cls(response)
-        :rtype: None
+        :returns: None, or the result of cls(response)
+        :rtype: None or Dict[str, Union[str, datetime]]
```

Comment:
This is generated code so we can't change it. Also, the return type of `None` is sort of correct here since it does return None unless a cls is provided.

Reply:
Doh. Too hasty on the F12 and diving straight into the docs, should be more weary of making sure I'm not touching `_generated`. Will do!